lnrpc/lightning.proto

```diff
@@ -1678,6 +1678,9 @@ message GetInfoResponse {
     announcements and invoices.
     */
     map<uint32, Feature> features = 19;
+
+    // Whether the lnd server has been fully started.
+    bool started = 21;
```

Comment:
Why do we need this if we have the main status endpoint? 

It's also the case that there exists rpc methods that _can't_ be called before the server is fully started up. The work to ensure we send out specific error codes could help here as well.

Reply:
Which main status endpoint? The `InterceptorChain.GetState`?
Initially I did think about putting the state there, only to found out later the `GetState` seemed to be referring to wallet state, not the lnd server state. On the other hand, the rpc server is started before the main server, and we only `SetRPCActive()` right after the rpc server is started.

I think what we could do instead is to add a new state to `InterceptorChain`, and a new method `SetServerActive()` to mark the server is fully started.

Reply:
Yeah I mean either `GetState` or `SubscribeState`. I see what you mean in taht we only have RPC active, and not a state that triggers right after the server is fully active. I think going with the option of adding this new state is the way to go, we can then call this right after we do the final initialization to start up the server.

Reply:
Cool, changed to be using the interceptor chain, PTAL.